mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -139,19 +139,26 @@ private void calculateChanges(File projectDirectory, Configuration configuration
     }
 
     private void configureExtension(MavenSession session, Configuration configuration) {
-        logger.info("Enabling extension.");
         final MavenProjectConfigurator mavenProjectConfigurator = new MavenProjectConfigurator(configuration);
         final File dumpedConfigFile = configuration.dump(Paths.get("").toFile());
         session.getAllProjects().forEach(mavenProject -> {
-            mavenProjectConfigurator.configureTestRunner(mavenProject.getModel());
-            copyConfigurationFile(mavenProject.getModel(), dumpedConfigFile);
-            if (isFailedStrategyUsed()) {
-                SurefireReportStorage.copySurefireReports(mavenProject.getModel());
+            MavenPropertyResolver mavenPropertyResolver = new MavenPropertyResolver(mavenProject.getModel());
+            if (mavenPropertyResolver.isSkipTestsSetInPom()) {
+                logger.debug("Disabling Smart Testing %s in %s module. Reason: Test Execution has been skipped.",
+                    ExtensionVersion.version().toString(), mavenProject.getArtifactId());
+
+            } else {
+                logger.info("Enabling extension.");
```

Comment:
This will be printed for every module, which is not expected. Bring it back as the extension is enabled anyway, then it just goes through the modules and enables/disables for every module separately